examples/msgpack/msgpack_entity.go

```diff
@@ -0,0 +1,42 @@
+package restPack
+
+import (
+	restful "github.com/emicklei/go-restful"
+	"gopkg.in/vmihailenco/msgpack.v2"
+)
+
+const MIME_MSGPACK = "application/x-msgpack" // Accept or Content-Type used in Consumes() and/or Produces()
+
+// NewEntityAccessorMPack returns a new EntityReaderWriter for accessing MessagePack content.
+// This package is not initialized with such an accessor using the MIME_MSGPACK contentType.
+func NewEntityAccessorMsgPack(contentType string) restful.EntityReaderWriter {
+	return entityMsgPackAccess{ContentType: contentType}
```

Comment:
do you really need the ContentType to be configurable?  The XML version needed this field to specify character encoding. You probably use the fixed const.